src/forms/todoForm.js

```diff
@@ -0,0 +1,243 @@
+
+
+  const priorityHigh = document.createElement('option');
+  priorityHigh.setAttribute('value', 'High');
+  priorityHigh.textContent = 'High';
+  priorityInput.appendChild(priorityHigh);
+
+
+  const priorityMedium = document.createElement('option');
+  priorityMedium.setAttribute('value', 'Medium');
+  priorityMedium.textContent = 'Medium';
+  priorityInput.appendChild(priorityMedium);
+
+
+  const priorityLow = document.createElement('option');
+  priorityLow.setAttribute('value', 'Low');
+  priorityLow.textContent = 'Low';
+  priorityInput.appendChild(priorityLow);
+
+
+  const descriptionDiv = document.createElement('div');
+  descriptionDiv.className = 'descriptinDiv';
+  form.appendChild(descriptionDiv);
+
+
+  const descriptionLabel = document.createElement('label');
+  descriptionLabel.className = 'block tLabel text-gray-700';
+  descriptionLabel.setAttribute('for', 'description');
+  descriptionDiv.appendChild(descriptionLabel);
+
+
+  const descriptionInput = document.createElement('textarea');
+  descriptionInput.className = 'mt-0 rounded-lg block w-full px-0.5 border-0 border-b-2 border-gray-200 focus:ring-0 focus:border-black';
+  descriptionInput.setAttribute('type', 'text');
+  descriptionInput.setAttribute('name', 'description');
+  descriptionInput.setAttribute('row', '3');
+  descriptionInput.setAttribute('id', 'todo-description');
+  descriptionDiv.appendChild(descriptionInput);
+
+  const descriptionWarning = document.createElement('small');
+  descriptionWarning.className = 'descriptionWarning';
+  descriptionWarning.style.display = 'none';
+  descriptionDiv.appendChild(descriptionWarning);
+
+
+  const submitBtn = document.createElement('button');
+  submitBtn.setAttribute('type', 'submit');
+  submitBtn.setAttribute('id', 'submitBtn');
+  submitBtn.textContent = index ? 'Edit' : 'Submit';
+  submitBtn.className = 'px-8 py-4 text-sm text-purple-600 font-semibold rounded-full border border-purple-200 hover:text-white hover:bg-purple-600 hover:border-transparent focus:outline-none focus:ring-2 focus:ring-purple-600 focus:ring-offset-2';
+  form.appendChild(submitBtn);
+
+  const closeBtn = document.createElement('button');
+  closeBtn.innerHTML = 'Cancel';
+  closeBtn.className = 'px-8 py-4 text-sm text-purple-600 font-semibold rounded-full border border-purple-200 hover:text-white hover:bg-purple-600 hover:border-transparent focus:outline-none focus:ring-2 focus:ring-purple-600 focus:ring-offset-2';
+  form.appendChild(closeBtn);
+
+
+  function deleteForm2(e) {
+    if (e.key === 'Escape') {
+      container.remove();
+    }
+
+    document.removeEventListener('keydown', deleteForm2);
+  }
+
+
+  function deleteForm(e) {
+    container.remove();
+    e.target.removeEventListener('click', deleteForm);
+    document.removeEventListener('keydown', deleteForm2);
+  }
+
+  function showWarning(element) {
+    element.style.display = 'block';
+
+    setTimeout(() => {
+      element.style.display = 'none';
+    }, 6000);
+  }
```

Comment:
- [ ] Also kindly change these to ES6 arrow function syntax. Please make the change in other affected functions too.